File: app/src/main/java/br/com/popularmovies/movies/data/response/Movies.java
package br.com.popularmovies.movies.data.response;


import com.squareup.moshi.Json;

import java.util.List;

public class Movies {

    @Json(name = "results")
    private List<Movie> movies;

    public List<Movie> getMovies() {
        return movies;
    }
}

File: datasource_remote/src/main/java/br/com/popularmovies/datasourceremote/models/movie/Movies.kt
package br.com.popularmovies.datasourceremote.models.movie

import android.os.Parcelable
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.android.parcel.Parcelize

@Parcelize
@JsonClass(generateAdapter = true)
data class Movies(@Json(name = "results") val movieDtos: List<MovieDto>) : Parcelable